dashboard/src/app/workspaces/list-workspaces/workspace-item/workspace-item.html

```diff
@@ -53,15 +56,15 @@
            class="che-list-item-name workspace-item-ram"
            ng-click="workspaceItemCtrl.redirectToWorkspaceDetails();">
         <span class="che-xs-header noselect" hide-gt-xs>RAM</span>
-        <span class="workspace-consumed-value">{{workspaceItemCtrl.getMemoryLimit(workspaceItemCtrl.workspace)}}</span>
+        <span class="workspace-consumed-value" name="workspaceRamValue">{{workspaceItemCtrl.getMemoryLimit(workspaceItemCtrl.workspace)}}</span>
```

Comment:
Why don't we user "**id**" attribute here instead of "**name**"?

Reply:
This RAM value exists for every workspace in list.

Reply:
I see, thank you!

Reply:
is that name shouldn't follow convention of Kebab case vs camelCase like for id where we have "-" ?